### PR TITLE
[#537] Update participant signature

### PR DESCRIPTION
**Pull request by jhflorey**
this PR addresses some comments from PR https://github.com/episphere/connectApp/issues/537

- Updated participant signature to address comment https://github.com/episphere/connectApp/issues/537#issuecomment-1666025860
- Added the suffix to participant signature to address comment https://github.com/episphere/connectApp/issues/537#issuecomment-1666137280

### CODE REVIEW

**Review by abhinavjonnada82 — CHANGES_REQUESTED** [review]

File: js/pages/agreements.js [hunk 1]
```diff
@@ -325,8 +325,9 @@ const addEventAgreementOptions = (myData) => {
 }
 
 const renderDownloadRevoke = async (data) => {
-    const middleName = data[436680969] ? ` ${data[436680969]} ` : ' ';
-    const participantSignature = `${data[471168198]}${middleName}${data[736251808]}`;
+    const middleName = data[826240317] ? ` ${data[826240317]} ` : ' ';
+    const suffix = data[693626233] || '';
```
[the remark this answers is not in the document]
Comment (jhflorey) [reply js/pages/agreements.js:329]: Got it. Thanks @abhinavjonnada82 @jhflorey

File: js/pages/agreements.js [hunk 2]
```diff
@@ -542,7 +545,8 @@ async function generateSignedPdf(data, file) {
   const siteDict = siteAcronyms();
   const participantSite = siteDict[data['827220437']];
   const middleName = data[436680969] ? ` ${data[436680969]} ` : ' ';
-  const participantFullName = `${data[471168198]}${middleName}${data[736251808]}`;
+  const suffix = data[480305327] ? suffixToTextMap.get(parseIntdata[480305327]) : '';
+  const participantFullName = `${data[471168198]}${middleName}${data[736251808]} ${suffix}`;
```
[the remark this answers is not in the document]
Comment (abhinavjonnada82) [reply js/pages/agreements.js:549]: ` `${data[471168198]}${middleName}${data[736251808]} ${suffix}.trim()``

**Comment by jhflorey** [comment]
@JoeArmani already updated and tested on my local.

**Review by JoeArmani — APPROVED** [review]
looks good

**Review by anthonypetersen — APPROVED** [review]
looks good

**Pull request merged**